jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/settlerimages/SettlerImageMap.java

```diff
@@ -182,18 +168,18 @@ private EDirection parseDirection(final String directionString) {
 		return direction;
 	}
 
-	private int calculatePriority(EMovableType type, EMovableAction action, EMaterialType material, EDirection direction) {
+	private int calculatePriority(SettlerImageFlavor settlerImageFlavor) {
```

Comment:
Move this method to SettlerImageFlavor, since it only accesses fields of the passed object.